I0517 03:08:38.457665 140238090389312 run.py:28] Device: cuda
I0517 03:08:39.882828 140238090389312 diagnostics.py:438] 167/3432 coordinates fall outside of the mask. Removing them.
I0517 03:08:41.705517 140238090389312 diagnostics.py:438] 163/4704 coordinates fall outside of the mask. Removing them.
I0517 03:08:42.251922 140238090389312 diagnostics.py:438] 12/1017 coordinates fall outside of the mask. Removing them.
I0517 03:08:42.828645 140238090389312 diagnostics.py:438] 67/1119 coordinates fall outside of the mask. Removing them.
I0517 03:08:47.303565 140238090389312 diagnostics.py:438] 313/12488 coordinates fall outside of the mask. Removing them.
/well/nichols/users/pra123/anaconda3/envs/torch/lib/python3.11/site-packages/patsy/util.py:672: FutureWarning: is_categorical_dtype is deprecated and will be removed in a future version. Use isinstance(dtype, CategoricalDtype) instead
  return _pandas_is_categorical_dtype(dt)
/well/nichols/users/pra123/anaconda3/envs/torch/lib/python3.11/site-packages/patsy/util.py:672: FutureWarning: is_categorical_dtype is deprecated and will be removed in a future version. Use isinstance(dtype, CategoricalDtype) instead
  return _pandas_is_categorical_dtype(dt)
